waffle/tests/test_management.py

```diff
@@ -28,6 +28,17 @@ def test_create(self):
         self.assertEqual(list(flag.groups.values_list('name', flat=True)),
                          ['waffle_group'])
 
+    def test_not_create(self):
+        """ The command shouldn't create a new flag if the create flag is
+        not set.
+        """
+        name = 'test'
+        self.assertFalse(Flag.objects.filter(name=name).exists())
```

Comment:
The assertions should come _after_ the calls to the management command. The purpose of the test is to ensure the command doesn't create a new instance.

Reply:
aha, I misunderstood your previous comment